html/captive-portal/templates/wireless-profile-noeap.xml

```diff
@@ -9,7 +9,11 @@
                         <key>AutoJoin</key>
                         <true/>
                         <key>EncryptionType</key>
+                        [% IF for_ios && provisioner.security_type == 'WPA' %]
```

Comment:
Perhaps the TT syntax supports both but we usually use `AND`

Example: https://github.com/inverse-inc/packetfence/blob/devel/html/captive-portal/templates/wireless-profile-peap.xml#L89

If it works, then it's all good. Otherwise, let's use the other common syntax